packages/test/test-end-to-end-tests/src/test/SummarizeFetchValidation.spec.ts

```diff
@@ -110,7 +110,7 @@ async function createSummarizer(
  */
 describeCompat(
 	"Summarizer fetches expected number of times",
-	"NoCompat",
+	"2.0.0-rc.1.0.0",
```

Comment:
```suggestion
	"2.0.0-rc.1.0.0" /* minVersion */,
```
nit: Add this and a comment explaining what it is. If we do this to the first few tests, it will carried over to other tests and it will help others understand and use it correctly.